src/app/common/ResourceCodeBlock.vue

```diff
@@ -76,3 +87,15 @@ function toYamlRepresentation(resource: Entity): string {
   return toYaml(resourceWithoutTimes)
 }
 </script>
+
+<style lang="scss">
```

Comment:
If these aren't scoped, should we manually scope them by giving this component a class of `resource-code-block`?

Reply:
I don’t see a need to make the scoping of the selectors in use here more tight. They’re specific enough to not risk conflicts.